src/Parsers/ParserSystemQuery.cpp

```diff
@@ -324,6 +324,7 @@ bool ParserSystemQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Expected &
         case Type::STOP_DISTRIBUTED_SENDS:
         case Type::START_DISTRIBUTED_SENDS:
         case Type::UNLOAD_PRIMARY_KEY:
+        case Type::LOAD_PRIMARY_KEY:
```

Comment:
```cpp
        case Type::LOAD_PRIMARY_KEY:
        case Type::UNLOAD_PRIMARY_KEY:
```

Reply:
Fixed

Reply:
I still see the `LOAD_PRIMARY_KEY` after the `UNLOAD_PRIMARY_KEY`